src/io/PipeWriter.zig

```diff
@@ -394,8 +394,8 @@ pub fn PosixStreamingWriter(
         is_done: bool = false,
         closed_without_reporting: bool = false,
 
-        // TODO:
-        chunk_size: usize = 0,
+        // TODO: configurable?
+        const chunk_size: usize = 1024 * 4;
```

Comment:
should be configurable, though it doesn't have to block this PR.

node:fs streams have this option. we can add an options argument into Bun.file().writer()

https://nodejs.org/api/fs.html#filehandlecreatewritestreamoptions